prime-router/docs/changelog-vs-release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Changelog vs Release Notes
+
+The [release notes](./release-notes.md) are a high-level summary of changes to ReportStream over time. 
```

Comment:
Maybe make the distinction here that release notes are customer facing changes and features, while the changelog includes all technical changes.

Reply:
yep thanks!